docs/tutorials/notebooks/Monge_Gap.ipynb

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
<p>Feels like the first "big" equation (in the begin{equation} env) is not rendering in this NB.</p><p> </p><p>squared-Euclidean (capital letter)</p><p> </p><p>Explain in what sense this provides an alternative to the <code>solver = Sinkhorn(norm_error=1)</code> solver, and, ideally, we might want to run it in this solver too? </p><p> </p><p>When defining the monge gap, mention to <code>cost_fn</code> is not provided. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>